.github/workflows/ExpressionInSync.yml

```diff
@@ -9,6 +9,6 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Download Expression.g4 from RedHatInsights/policies-engine
-      run: curl -O https://raw.githubusercontent.com/RedHatInsights/policies-engine/master/api/src/main/antlr4/com/redhat/cloud/policies/api/model/condition/expression/parser/Expression.g4
+      run: curl -O https://raw.githubusercontent.com/RedHatInsights/policies-engine/master/src/main/antlr4/com/redhat/cloud/policies/engine/condition/Expression.g4
```

Comment:
I didn't think this would be used in the frontend.

Reply:
It's used to generate the antlr code that allows autocompletion - not really used in all its power but allows some basics syntax.